source/classes.tex

```diff
@@ -2643,7 +2643,7 @@
 
 void h(Z z) {
   Y y1(z);          // OK: direct-initialization
-  Y y2 = z;         // ill-formed: copy-initialization
+  Y y2 = z;         // error: copy-initialization
```

Comment:
// error: no candidate conversion functions for copy-initialization

or something like that?

Reply:
Fixed.